lib/ohai/plugins/windows/features.rb

```diff
@@ -0,0 +1,20 @@
+Ohai.plugin(:Features) do
+  provides "installed_features"
+  collect_data(:default) do
+    installed_features Mash.new
+    # Grab raw feature information
+    raw_list_of_features = shell_out("C:\\Windows\\sysnative\\dism.exe /Get-Features /Online /Format:Table").stdout
```

Comment:
You should probably grab the Windows directory location from %WINDIR% in the case where Windows is installed to a non "c" drive.